demo/site/package.json

```diff
@@ -16,7 +16,7 @@
         "lint:eslint": "eslint --max-warnings 0 --config ./.eslintrc.cli.js --ext .ts,.tsx,.js,.jsx,.json,.md src/ package.json",
```

Comment:
```suggestion
        "lint:eslint": "eslint --max-warnings 0 --config ./.eslintrc.cli.js --ext .ts,.tsx,.js,.jsx,.json,.md src/ package.json server.ts tracing.ts",
```

Reply:
hm, I don't like having to list all files here, can't we use ./ instead of src/?

Reply:
Then we'll have to put all non-lintable files in a .eslintignore

Reply:
Do you want to avoid that?

Reply:
No, I just wanted to point out that we'll have to keep this list somewhere. But .eslintignore is probably better

Reply:
I will address that in a different PR, this is out of scope (the old non-typed files where also not linted)

Reply:
https://github.com/vivid-planet/comet/pull/2600